jest.setup.js

```
@@ -42,6 +42,20 @@ jest.mock('react-native-file-viewer', () => ({
 
 jest.mock('expo-haptics', () => jest.fn(() => null));
 
+jest.mock('react-native-gesture-handler', () => {
+	const React = require('react');
+	const { View } = require('react-native');
+	const GestureHandlerRootView = React.forwardRef(({ children, ...props }, ref) => (
+		<View ref={ref} {...props}>{children}</View>
```

ESLint

[error] Replace `{children}` with `⏎↹↹↹{children}⏎↹↹`

(prettier/prettier)